scripts/generate_json_docs.py

```diff
@@ -290,10 +295,10 @@ def build_type(type_id, title, contents):
 
 def clean_source_path(source):
     source_path = ''
-    if 'gcloud-python' in source:
-        source_path = source.split('gcloud-python')[1]
+    if ABSOLUTE_LIBRARY_PATH in source:
+        source_path = source.replace(ABSOLUTE_LIBRARY_PATH, '')
```

Comment:
@tseaver, I can just have it default to `source` if you want and we can see if it causes any issues down the line.